iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/BusExt.java

```diff
@@ -16,8 +16,10 @@
  */
 interface BusExt extends Bus {
 
+    @Override
```

Comment:
To remove as it is already in Bus?

Reply:
Leaving it forced the classes inheriting BusExt to use TerminalExt, and not another class extending Terminal.

Reply:
For instance, with latest commit removing it, MergedBus.getConnectedTerminals() is now returning Iterable<Terminal> and not Iterable<TerminalExt>, and dealing with ConfiguredBus.getConnectedTerminals() hence with terminals whose type is `? extends Terminal`.